foreman-selinux-relabel

```diff
@@ -11,8 +11,11 @@
   /etc/puppetlabs/puppet/node.rb \
   /etc/sysconfig/foreman* \
   /etc/rc.d/init.d/foreman* \
+  /usr/lib/systemd/system/foreman* \
+  /etc/systemd/system/foreman* \
   /etc/logrotate.d/foreman* \
   /etc/cron.d/foreman* \
+  /usr/libexec/foreman \
   /usr/lib{64,}/ruby/gems/1.8/gems/passenger-* \
```

Comment:
This can be dropped since it's EL6 code. Same for all other Ruby 1.8 stuff. Feel free to do that in a later PR.

Reply:
Yeah I know the policy is in bad shape, I wanted to to "no changes commit" first since we are removing it pretty soon. I can do a followup commit where I simply remove it all and we would only cherry pick the first one into 2.1. Lemme see your other comments first.

Reply:
I don't know if you mean just this line or all of the lib64 ones, I'd keep it until we remove all of the passenger code in a separate PR if you don't mind.

Reply:
The entire line can go. `/usr/lib{64,}/ruby/gems/1.8` existed on EL6 which had Ruby 1.8.7 but luckily we're beyond those dark days.